src/basins/clustering/cluster_config.jl

```diff
@@ -15,8 +15,8 @@ which is unsupervised, see Description below.
 
 ## Keyword arguments
 * `templates = nothing`: Enables supervised version, see below. If given, must be a
-  vector of `Vector`s, each inner vector containing the features representing a center of a
-  cluster.
+   Dictionary of `Vector`s, each containing the features representing a cluster (which can
```

Comment:
```suggestion
   dictionary mapping cluster labels to `Vector`s, each containing the features representing a cluster (which can
```
a dictionary has keys and values, so a dictionary of vectors is ambiguous!

Reply:
good point, or maybe then
"Dictionary of cluster labels to cluster features. The labels can be of arbitrary type, and the features are `Vector`s, representing a cluster (which can be an attractor, for  instance)."

Reply:
Perfect!